natlas-server/app/scope/scope.py

```diff
@@ -92,7 +93,8 @@ def updateScanManager(self):
 			scanrange = [IPNetwork(n.target) for n in ScopeItem.getScope()]
 			blacklistrange = [IPNetwork(n.target) for n in ScopeItem.getBlacklist()]
 			self.scanmanager = IPScanManager(scanrange, blacklistrange)
-		except Exception:
+		except Exception as e:
```

Comment:
Can we improve this exception handling? Whatever exception happens that leads to "No scope configured" should be logged, but if any other exceptions happen then we want to let them raise up to be captured by sentry or otherwise print to the console. I know that this is in code I wrote, but unfortunately now that you've edited the line I get to ask you to improve it 😅